test/SymbolGraph/ClangImporter/ErrorEnum.swift

```diff
@@ -18,6 +18,12 @@
 // but it should be hidden for a similar reason
 // CHECK-NOT: "precise": "s:SYsSHRzSH8RawValueSYRpzrlE4hash4intoys6HasherVz_tF::SYNTHESIZED::c:@E@MyErrorCode"
 
+// MyError
+// CHECK-DAG: "precise": "s:SC11MyErrorCodeLeV"
+
+// MyError.errFirst
+// CHECK-DAG: "precise": "s:SC11MyErrorCodeLeV8errFirstSoAAVvpZ"
```

Comment:
Thanks for updating this test! This would have been my only concern with this PR, but it looks like that's settled.